app/views/posts/show.html.erb

```diff
@@ -1 +1,27 @@
-<h1>Post Details</h1>
\ No newline at end of file
+<br/>
+<div class="container-sm">
+  <div class="card border-success mb-3">
+    <div class="card-body text-success">
+      <div class="d-flex justify-content-between">
+        <div>
+          <h4><span class="card-title">Post #<%= @post_id %></span> <span>by <%= @post.author.name %></span></h4>
+        </div>
+        <div>
+          <span class="card-text">Comments: <%= @post.comments_counter%></span>
+          <span class="card-text">Likes: <%= @post.likes_counter%></span>
+        </div>
+      </div>
+      <p class="card-text"><%= @post.text %></p>
+      <div class= "card-text">
+        <hr>
+        <div>
+          <% @post.comments.each do |comment|%>
+            <p class = "m-0"><strong><%= comment.author.name%></strong>: 
+              <%= comment.text%>
+            </p>
+          <% end %>
+        </div>
```

Comment:
- [ ] Amazing job working on the views for users but when I try to press on all posts to be able to see index page with all posts it shows me this error as well so kindly make sure to solve the issues so I can be able to see all the views you have created ✨⭐🚀💪🏻

![image](https://github.com/noelfoka/Blog_app/assets/58553711/2cb56a16-db66-421b-b96e-bea7c9601744)



- I should see something like this 
![image](https://github.com/noelfoka/Blog_app/assets/58553711/bd5c04ad-3a70-4434-870d-929aeb931d62)

Reply:
This issue has been resolved